src/current/cockroachcloud/quickstart.md

```diff
@@ -19,6 +19,8 @@ This page shows you how to use the [{{ site.data.products.db }} Console](https:/
 
 ## Connect to the cluster
 
+Select a language to connect a sample application to your cluster. To connect to your cluster directly from the command line, refer to [Connect to a {{ site.data.products.serverless }} Cluster](connect-to-a-serverless-cluster.html?fiters=command-line).
```

Comment:
Should the link go directly to this section: https://deploy-preview-17646--cockroachdb-docs.netlify.app/docs/cockroachcloud/connect-to-a-serverless-cluster?fiters=command-line#connect-to-your-cluster